llm-disease-categorization/conf/base/parameters.yml

```diff
@@ -0,0 +1,73 @@
+categorization_prompt: Identify and return ONLY (provide no additional text) the best-fitting category of the following list of categories for the disease
+
+disease_categories_txgnn_modified:
+  name: txgnn
+  categories:
+    - adrenal_gland_disease
+    - neurodegenerative_disease
+    - metabolic_disorder
+    - cardiovascular_disorder
+    - autoimmune_diseases
+    - mental_health_disorder
+    - anemia
+    - cancer
+    - inflammatory_disease
+    - allergy
+    - other
+
+disease_categories_anatomical:
+  name: anatomical
```

Comment:
Is there a source for these?

Reply:
Nope! Just came up with them based on feedback from Elliott.